hybridse/src/udf/udf_registry.h

```diff
@@ -1116,12 +1116,52 @@ struct TypeAnnotatedFuncPtrImpl<std::tuple<Args...>> {
     GetTypeF get_ret_type_func;
 };
 
+template <typename Ret, typename... Args>
+struct TypeAnnotatedFuncPtrImpl<Ret(Args...)> {
+    using GetTypeF =
+        typename std::function<void(node::NodeManager*, node::TypeNode**)>;
+
+    // TypeAnnotatedFuncPtr can only be bulit from non-void return type
```

Comment:
return type info will be inferred by `get_return_type_func` for void functions, maybe that's because `DataTypeTrait` do not support tuple ?